### PR TITLE
Add sentence for second person to style guide

### PR DESCRIPTION
**Pull request by knylander-grafana**
We aren't clear enough that we use second person. The only place that it's clearly stated with examples is in the release notes section. I've used that same sentence in the style guide page under how to address users.

### CODE REVIEW

**Comment by josmperez** [comment]
You're saying that the "you" is always implied. So is a procedure wrong in your view if the "you" is explicit. as in, "The server is powered off. You can turn it on by pressing the power button."

**Comment by jdbaldry** [comment]
Best guidance I have for this right now is in the Vale rule that has triggered on this PR for the use of "we".

If we write a more clear explanation, I'd like to update that rule to link to whatever section that guidance ends up in.

**Comment by jdbaldry** [comment]
> You're saying that the "you" is always implied. So is a procedure wrong in your view if the "you" is explicit. as in, "The server is powered off. You can turn it on by pressing the power button."
> 
> IOW, should we be rewriting explicit yous to implicit yous throughout all our docs?

Are you referring to https://github.com/grafana/writers-toolkit/pull/865/files#diff-dce4785db28c782d8c85d67948ddaed3a01acc26b986f31c08fc02e666f9294fR40 which says "To address users clearly and directly, write directives in the imperative second person where the unspoken 'you' is implied."

It does seem that we would prefer "The server is powered off. Turn it on by pressing the power button."

**Comment by knylander-grafana** [comment]
> Are you referring to [#865 (files)](https://github.com/grafana/writers-toolkit/pull/865/files#diff-dce4785db28c782d8c85d67948ddaed3a01acc26b986f31c08fc02e666f9294fR40) which says "To address users clearly and directly, write directives in the imperative second person where the unspoken 'you' is implied."
> 
> It does seem that we would prefer "The server is powered off. Turn it on by pressing the power button."

I think this is definitely preferred when you are referring to procedure steps. However, when you're talking about concepts or reference, the imperative might not make sense. 

I've updated conceptual docs so many times and requested not to use the first person. I'll rework this to not use "Our" in the sentence. 

I miss the "imperative second person" because the keyword that I'm looking for is usually "we".

File: docs/sources/write/style-guide/style-conventions/index.md [hunk 1]
```diff
@@ -35,7 +35,10 @@ There are some situations where you must mention another company, such as in dat
 
 ## Address users clearly
 
+Our style prefers second person ("you") and avoids first person ("I"/"we").
+
 To address users clearly and directly, write directives in the imperative second person where the unspoken 'you' is implied.
+
 Avoid making the user or their role, such as _system administrator_, the subject of a sentence, as in: "Users configure the Cloud using a secure shell.".
```
Comment (knylander-grafana) [inline docs/sources/write/style-guide/style-conventions/index.md:42]:
```suggestion
<!-- vale Grafana.We = NO -->
To address users clearly and directly, use second person ("you") and avoid first person ("I" or "we"). 
Write directives in the imperative second person where the unspoken 'you' is implied.
<!-- vale Grafana.We = Yes -->

Avoid making the user or their role, such as _system administrator_, the subject of a sentence, as in: "Users configure the Cloud using a secure shell.".
```

Comment (jdbaldry) [reply docs/sources/write/style-guide/style-conventions/index.md:42]: That sounds good to me!

File: docs/sources/write/style-guide/style-conventions/index.md [hunk 2]
```diff
@@ -35,7 +35,15 @@ There are some situations where you must mention another company, such as in dat
 
 ## Address users clearly
 
-To address users clearly and directly, write directives in the imperative second person where the unspoken 'you' is implied.
+<!-- vale Grafana.We = NO -->
+To address users clearly and directly, use second person ("you") and avoid first person ("I", "our", or "we"). 
+Use first person plural pronouns like 'our' carefully.
+Don't use 'we' when you're talking about the reader, instead use 'you'.
+It's OK to use 'we' when you're talking about Grafana Labs.
```
[the remark this answers is not in the document]
Comment (knylander-grafana) [reply docs/sources/write/style-guide/style-conventions/index.md:42]:
```suggestion
You can use 'we' when you're talking about Grafana Labs.
```

File: docs/sources/write/style-guide/style-conventions/index.md [hunk 3]
```diff
@@ -35,7 +35,15 @@ There are some situations where you must mention another company, such as in dat
...
+<!-- vale Grafana.We = Yes -->
```
[the remark this answers is not in the document]
Comment (jdbaldry) [reply docs/sources/write/style-guide/style-conventions/index.md:45]:
```suggestion
<!-- vale Grafana.We = YES -->
```

**Review by jdbaldry — APPROVED** [review]

**Pull request merged**